connectors/preflight_check.py

```diff
@@ -49,6 +54,35 @@ async def run(self):
             if self.es_client is not None:
                 await self.es_client.close()
 
+    async def _check_local_extraction_setup(self):
+        if not self.extraction_config.get("enabled", False):
+            return
+
+        timeout = aiohttp.ClientTimeout(total=5)
+        session = aiohttp.ClientSession(timeout=timeout)
+        try:
+            async with session.get(
+                f"{self.extraction_config['host']}/ping/"
+            ) as response:
+                if response.status != 200:
```

Comment:
Could there also be cases, where status codes `200 < response.status < 300` also should be considered successful?

Reply:
Thanks for the explanation, makes sense 👍